### ISSUE TITLE
User login/registration system available?

### ISSUE THREAD

**Issue by lovebdsobuj** [havok89/Hoosk#14]

**Comment by havok89** [comment]
There isnt one available at the moment but since it is in CodeIgniter it would be relatively simple to build one. how would you be using it?

when the user logs in would there be a seperate user dashboard, what features would it have etc

**Comment by lovebdsobuj** [comment]
Yes, when the user logs in would there be a separate user dashboard.
Feature,
- Profile
- Posts (only permitted id)
- Comments

etc...

Some page/post will be only for registered user.
Leave a comment (only for registered user).

**Comment by havok89** [comment]
when a user isnt logged in on the front end should they still see the pages on the navigation bar except when clicked they would say please log in to view this page?

**Comment by lovebdsobuj** [comment]
Yes, some page will be privileges only for logged user.

**Comment by PowerChaos** [comment]
at this moment there is no permission system , so new users ( manual added) are root by default

adding a permission tab would be nice , so i like the following roles added

Admin
|-> Root user , full permition

Designers
|-> Ability to only change the design , after confirm of a admin it will update the edited pages ( optional)

Posters
|-> Able to create blogposts and manage comments ( also known as moderator) 

feedback
|-> Normal registered users that can post comments on the posts, after a mod confirmed it (optional ) 

ofcourse those groups are not with recursive permission , so a poster can not design and a designer can not post

atleast this is my vision xD
Greets From PowerChaos

ps: instead using md5 for passwords , could you see if you could implent the following script for passwords ? https://github.com/defuse/password-hashing

Greets From PowerChaos
